monitor/visualizer.py

```diff
@@ -0,0 +1,95 @@
+import json
+import requests
+
+from monitor.searcher import LOG_PREFIX
```

Comment:
monitor.searcher does not exists. Watch out if you try to import it from monitor_start since that'll be a circular dependency and the code will fail (the error message in this cases is pretty vague and tricky t figure out xD)